tests/unit_test.py

```diff
@@ -150,6 +155,10 @@ def test_defining_conflicting_alias_raises():
     sc.units.aliases['speed'] = 'm/s'
     with pytest.raises(ValueError):
         sc.units.aliases['fastness'] = 'm/s'
+    with pytest.raises(ValueError):
+        sc.units.aliases['fastness'] = '100*cm/s'
+    with pytest.raises(ValueError):
+        sc.units.aliases['fastness'] = sc.scalar(1000, unit='mm/s')
```

Comment:
Not sure I get this test:
- Overriding alias (same name) should be fine?
- Different alias for same unit should raise.
- I can't see a test for different units with same base dims (but different scales) with different aliases. Having alias es for `1.2 m` and `1.4 m` at the same time should be possible, no?

Reply:
> Not sure I get this test:
> 
>     * Overriding alias (same name) should be fine?

Yes, see `test_can_override_alias`

> 
>     * Different alias for same unit should raise.

This is what this test checks.

> 
>     * I can't see a test for different units with same base dims (but different scales) with different aliases. Having alias es for `1.2 m` and `1.4 m` at the same time should be possible, no?

True, I can add that.